scripts/get-base-branch.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+# This script gets the base branch of the PR and is meant to be used by CircleCI jobs
+
+# CIRCLE_PULL_REQUEST is an environment variable set by CircleCI if a pull request exists
+# https://circleci.com/docs/variables/#built-in-environment-variables
+PR_NUMBER="${CIRCLE_PULL_REQUEST##*/}"
+
+if [[ -z "${PR_NUMBER}" ]]; then
+    echo "A PR number is required to get the base branch"
```

Comment:
I should know this, but don't we also build branches without PRs on CircleCI? What will happen when this script fails?